contrib/ros/envs/sawyer_env.py

```diff
@@ -0,0 +1,356 @@
+"""
+ROS environment for the Sawyer robot
+Every specific sawyer task environment should inherit it.
+"""
+
+import copy
+import os.path as osp
+
+import numpy as np
+import rospy
+import rospkg
+from gazebo_msgs.msg import ModelStates
+from geometry_msgs.msg import Pose, Point
+
+from rllab.core.serializable import Serializable
+from rllab.envs.base import Step
+from rllab.misc import logger
+from rllab.spaces import Box
+
+from contrib.ros.envs.ros_env import RosEnv
+from contrib.ros.robots.sawyer import Sawyer
+from contrib.ros.util.common import rate_limited
+from contrib.ros.util.gazebo import Gazebo
+
+import intera_interface
```

Comment:
this is a 3rd party dep, so it should be grouped along with numpy and ros deps above.